flink-kubernetes-operator-api/pom.xml

```diff
@@ -276,6 +276,14 @@ under the License.
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>11</source>
+                    <target>11</target>
+                </configuration>
+            </plugin>
```

Comment:
I don't understand why we make this change here.

Reply:
I added this code as prompted by the IDE, it may not be required and has been removed.

Reply:
Executing the mvn command locally verifies that it passes.

`mvn clean install -Pgenerate-docs`

<img width="1766" alt="image" src="https://user-images.githubusercontent.com/54518670/200539642-305f3458-950f-4f88-be5b-9e976fa4377b.png">
<img width="1191" alt="image" src="https://user-images.githubusercontent.com/54518670/200539773-be41dbbc-507c-4a20-a5a8-7e2cb944299e.png">

Reply:
what if you do git status after running the build?
I understand that it passes but it should not leave any new changes

Reply:
The CI checks that after running the doc generation there are no new changes in git, this is the check that fails

Reply:
Thanks ,this is git status after running the build.
<img width="1291" alt="image" src="https://user-images.githubusercontent.com/54518670/200547559-9e012d3d-2fb1-415d-812d-d0c12a80e913.png">

Reply:
Or, I turn off the PR and resubmit it again.

Reply:
> there should not be any more changed files after running
> 
> ```
> mvn clean install -Pgenerate-docs
> ```

This file has been changed.

`docs/content/docs/custom-resource/reference.md`